test/core/xds/xds_client_fuzzer.cc

```diff
@@ -57,7 +57,9 @@ class Fuzzer {
       // Leave xds_client_ unset, so Act() will be a no-op.
       return;
     }
-    auto transport_factory = MakeOrphanable<FakeXdsTransportFactory>();
+    auto transport_factory = MakeOrphanable<FakeXdsTransportFactory>([]() {
+      gpr_assertion_failed(__FILE__, __LINE__, "Multiple concurrent reads");
```

Comment:
Please use `GPR_ASSERT()` instead of expanding the macro.

Reply:
Done. I introduced a fake variable to make failure more descriptive